### DISCUSSION TITLE
Original width and height issue

### DISCUSSION THREAD

**Discussion by mageshjuhomi** [wojtekmaj/react-pdf#901] [Q&A] [answered]
[F-8879.pdf](https://github.com/wojtekmaj/react-pdf/files/7639932/F-8879.pdf)
I have been using  "react-pdf" library for our web application. I am facing an issue that the PDF file is not rendering in it's exact dimensions. I didn't mention any height, width and scale in the Page component.

So, I would like to know how we can get the exact dimensions (originalHeight & originalWidth) of our pdf.

For Instance,

When I open my PDF file in Adobe Acrobat Reader then it shows the size as 8.50 x 11.00 inches (i.e 816 x 1056 px). When I load the same pdf in my application using react-pdf then it renders by 612 x 792.008 px.

I have also tried the following onRenderSuccess function in Page component, but no luck.

```
const handleRenderSuccess = pageData => {
  console.log('width', pageData.width);
  console.log('height', pageData.height);
  console.log('originalWidth', pageData.originalWidth)
  console.log('originalHeight', pageData.originalHeight)
}
```

Results:

width 612
height 792.008
originalWidth 612
originalHeight 792.008

The above results are not returning values as per my expectation.

So, could anyone please help me out to find out the exact size of the PDF file using react-pdf library?

**![pdf_size](https://user-images.githubusercontent.com/47027787/144378401-76c29774-9224-4b29-8914-f4e1b375c057.png)**: Attached the Pdf file which I used for my test.

**Comment by wojtekmaj** [comment]
Huh, that's interesting. I opened this file in Mozilla Firefox and it rendered correctly at 816 x 1056px, despite dimensions being reported as 612 x 792.008px in `pageInfo`. This is exactly 75% of the actual rendered size, so it might have something to do with the DPI maybe? Mozilla's own PDF viewer seem to know something React-PDF does not know...

**Reply by mageshjuhomi** [reply]
> ### [ ](/wojtekmaj)

As you said, 612 x 792.008px is 75% of the actual rendered size. But, why this is happening? Can you please let me know where you are calculating this pdf size (612 x 792.008px) in your code? and how you are calculating dimensions of pdf file?

**Reply by wojtekmaj** [reply]
I don't know that. Will need to investigate.

**Comment by mageshjuhomi** [comment]
Is there a way to get 816 x 1056px using react-pdf?

**Comment by darkkeyper** [comment, marked as answer]
For anyone looking for the answer to this issue. The numbers posted above are correct but they aren't in pixels (px), they are in points (pt). There is 72pt per inch.  So if you do a quick bit of math 612 / 72 = 8.50 and 792 / 72 = 11.
